distro/superbuild/cmake/externals.cmake

```diff
@@ -224,9 +242,18 @@ endif()
 
 ###############################################################################
 # PythonQt
+set(PythonQt_REPO)
+set(PythonQt_TAG)
+if(${DD_QT_VERSION} VERSION_GREATER "4")
+  set(PythonQt_REPO https://github.com/patmarion/PythonQt.git)
+  set(PythonQt_TAG fix-compile-error-on-qt5.8)
```

Comment:
i think patched-7 has removed qt4 support. the src dir generated_cpp_47 was removed in that branch.